mitodl/open-discussions
# adds new field `availability` to Course model

#### What are the relevant tickets?
Closes #1898 in conjunction with PR #1899. 
Note: This PR can be reviewed and approved independently, but PR #1899 should be merged first.

#### What's this PR do?
Adds new `availability` field to `course_catalog/Course` model.

#### How should this be manually tested?
You could run the newly added migration and check if the `Course` table has the appropriate column added in Postgres via manual inspection, and that the values match the `raw_json`.
```python
from course_catalog.models import *
for c in Course.objects.all(): print(c.platform, c.availability)
```
